Modules/zlibmodule.c

```diff
@@ -1298,6 +1333,8 @@ static PyType_Slot Comptype_slots[] = {
     {0, 0},
 };
 
+// Using PyType_GetModuleState() on this type is safe since
+// it cannot be subclassed: it does not have the Py_TPFLAGS_BASETYPE flag.
```

Comment:
In your PR, you only pass the **defining class** to PyType_GetModuleState(), so it would be ok to subclass this type (even if it's not possible, since it's doesn't have Py_TPFLAGS_BASETYPE). You can remove this comment, and the copy below.